crates/aptos-telemetry-service/src/log_ingest.rs

```diff
@@ -28,6 +28,8 @@ pub fn log_ingest(context: Context) -> BoxedFilter<(impl Reply,)> {
             NodeType::Validator,
             NodeType::ValidatorFullNode,
             NodeType::PublicFullNode,
+            NodeType::UnknownFullNode,
```

Comment:
Silly question: why doesn't this contain `NodeType::Unknown`?

Reply:
Because, at this point Unknown is pretty much "deprecated" and exists only for backwards compatibility. We never ingested `Unknown` logs before, so it wasn't here.